### PR TITLE
fix: improve readability of node info metrics value

### LINKED ISSUE #6899: [Task]: Improve readability of node info metrics value
### Task description

The confirmation rate in the metrics of the node can be limited to 1 decimal. And it would be good to include a "%"

image

The suggestion came from #4516 

### Requirements

- [ ] Metrics are more readable

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

Closes #6899 

## Changelog

image

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/components/popups/NodeInfoPopup.svelte [hunk 1]
```diff
@@ -134,7 +138,13 @@
...
                         >{localize(`${NODE_INFO_LOCALE_BASE_PATH}.${localeKey}`)}</Text
                     >
-                    {#if typeof nodeInfoValue === 'boolean'}
+                    {#if localeKey === 'metrics.referencedRate' || localeKey === 'metrics.referencedBlocksPerSecond' || localeKey === 'metrics.blocksPerSecond'}
+                        <Text type="p" fontSize="sm" secondary>
+                            {formatMetricWithOneDecimal(nodeInfoValue)}{localeKey === 'metrics.referencedRate'
+                                ? '%'
+                                : ''}
+                        </Text>
+                    {:else if typeof nodeInfoValue === 'boolean'}
```
Comment (begonaalvarezd) [inline packages/desktop/components/popups/NodeInfoPopup.svelte:147]: I wouldnt add the logic here, there is a section specifically to preprocess the data, in `processNodeInfoMapTab`

You could do something like 

```ts
function processNodeInfoMapTab(
    _nodeInfoTab: NodeInfoTab,
    key: string
): { localeKey: string; nodeInfoValue: unknown } {
    const nodeInfoTabObject = NODE_INFO_TAB_MAP[_nodeInfoTab]
    const localeKey = nodeInfoTabObject[key].localeKey
    let nodeInfoValue = ''
    if (key === 'url') {
        nodeInfoValue = node?.url
    } else {
        nodeInfoValue = resolveObjectPath(nodeInfo, nodeInfoTabObject[key]?.nodeInfoPath, null)
        if (key === 'referencedRate') {
            let referencedRate = Number(nodeInfoValue)
            if (referencedRate >= 0) {
                nodeInfoValue = `${formatNumber(Math.min(referencedRate, 100), 1, 1)}%`
            } else {
                nodeInfoValue = ''
            }
        }
    }

    return {
        localeKey,
        nodeInfoValue,
    }
}
```
With this you shouldnt need any HTML changes nor the extra custom function `formatMetricWithOneDecimal` which feels also a bit redundant

File: packages/desktop/components/popups/NodeInfoPopup.svelte [hunk 2]
```diff
@@ -74,8 +74,20 @@
...
         const nodeInfoTabObject = NODE_INFO_TAB_MAP[_nodeInfoTab]
         const localeKey = nodeInfoTabObject[key].localeKey
-        const nodeInfoValue =
-            key !== 'url' ? resolveObjectPath(nodeInfo, nodeInfoTabObject[key]?.nodeInfoPath, null) : node?.url
+        let nodeInfoValue = ''
+        if (key === 'url') {
+            nodeInfoValue = node.url
+        } else {
+            nodeInfoValue = resolveObjectPath(nodeInfo, nodeInfoTabObject[key]?.nodeInfoPath, null)
+            if (localeKey === 'metrics.referencedRate') {
```
Comment (begonaalvarezd) [inline packages/desktop/components/popups/NodeInfoPopup.svelte:82]: I would use the key here and not the `localeKey`

```suggestion
            if (key === 'referencedRate') {
```

Comment (begonaalvarezd) [reply packages/desktop/components/popups/NodeInfoPopup.svelte:82]: I would also round to 1 or 2 decimals the 2 first elements
image

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
almost there 🌷

**Review by cpl121 — COMMENTED** [review]
When the rate information is copied, it appears in the form of a percentage (%), whereas when the entire information is copied, it appears in the previous format. I'm not sure if updating the `nodeInfo` store with the new format to maintain the same formatting would be correct

It's just a suggestion, as I'm not sure if what I'm saying is correct

**Comment by begonaalvarezd** [comment]
> When the rate information is copied, it appears in the form of a percentage (%), whereas when the entire information is copied, it appears in the previous format. I'm not sure if updating the `nodeInfo` store with the new format to maintain the same formatting would be correct
> 
> It's just a suggestion, as I'm not sure if what I'm saying is correct

I think its ok how it is now, we are beautifying it for presentational purposes but the copy information function uses a JSON that can be more technical and present the information directly fetched from the node

**Review by cpl121 — APPROVED** [review]
LGTM 🥇

**Review by begonaalvarezd — APPROVED** [review]
🚀

**Pull request merged**